planning/behavior_path_planner/include/behavior_path_planner/utilities.hpp

```diff
@@ -277,7 +277,9 @@ PathWithLaneId setDecelerationVelocity(
 
 bool checkLaneIsInIntersection(
   const RouteHandler & route_handler, const PathWithLaneId & ref,
-  const lanelet::ConstLanelets & lanelet_sequence, double & additional_length_to_add);
+  const lanelet::ConstLanelets & lanelet_sequence, double & additional_length_to_add,
+  const BehaviorPathPlannerParameters & parameters);
```

Comment:
@zulfaqar-azmi-t4 
Could you sort the order of the arguments so that the output comes last?

Reply:
Yes sure.
The parameters are now reordered.